frontend/js/models/annotation.js

```diff
@@ -46,15 +45,44 @@ define(
             defaults: function () {
                 return {
                     start: 0,
-                    duration: 0,
-                    comments: new Comments([], { annotation: this })
+
+                    // Value > 0 for time based annotation by default (contrary to point annotation)
+                    duration: 0.5,
+
+                    comments: new Comments([], { annotation: this }),
+                    content: new AnnotationContent([])
                 };
             },
 
+            /**
+             * Constructor
+             * @alias module:models-annotation.Annotation#initialize
+             * @param {object} attr Object literal containing the model initialion attributes.
+             */
+            initialize: function (attr) {
+                // Fixes comments not being loaded/updated anymore (e.g. reload app); also fixes icon status.
+                // This was removed in master (35e244e000f82ff8def250bfe566bd213d4705a2) - Check if this is correct.
```

Comment:
In master in questionnaire the comment loading behaviour seems a bit weird at this point, this should be checked in the review I guess